### ISSUE TITLE
Cactus 3.0.0 not available on PyPI

### ISSUE THREAD

**Issue by pmdarrow** [koenbok/Cactus#121]
```bash
~/Code $ pip install Cactus==3.0.0
Downloading/unpacking Cactus==3.0.0
  Could not find a version that satisfies the requirement Cactus==3.0.0 (from versions: 2.2.1, 2.3.0, 2.3.1)
  Some externally hosted files were ignored (use --allow-external to allow).
Cleaning up...
No distributions matching the version for Cactus==3.0.0
```

**Comment by krallin** [comment]
The issue here is that the package isn't hosted on PyPi itself, but GitHub.

@koenbok ,

Any specific reason you're not pushing the pack

**Comment by koenbok** [comment]
No specific reason. I didn't know pip disallowed external resources by default.

@krallin do you want me to fix it? Or shall I share the pypi login with you?

**Comment by krallin** [comment]
@koenbok ,

Either way works. 

PyPi should let you add me as a maintainer on Cactus without sharing any credentials (just login, open the Cactus PyPi page, click on Roles, and add me as a maintainer; my username is `Thomas.Orozco`).

Cheers,

**Comment by koenbok** [comment]
Done :-)

**Comment by krallin** [comment]
Fixed with 2b9cde653c07a9cf81f358c0122fd58a4e1ee166 (v3.0.1).

https://pypi.python.org/pypi/Cactus

Cheers,
